docs/pages/identity-governance/access-request-plugins/ssh-approval-email.mdx

```diff
@@ -117,10 +111,26 @@ longer-lived identity files with `tctl`:
 (!docs/pages/includes/plugins/tbot-identity.mdx secret="teleport-plugin-email-identity"!)
 </TabItem>
 <TabItem label="Long-lived identity files">
-(!docs/pages/includes/plugins/identity-export.mdx user="access-plugin" secret="teleport-plugin-email-identity" !)
+(!docs/pages/includes/plugins/identity-export.mdx user="access-plugin" secret="teleport-plugin-email-identity"!)
 </TabItem>
 </Tabs>
 
+## Step 4/7. Install the Teleport Email plugin
+
+<details>
```

Comment:
The positioning of the details box makes it a little hard to tell what's going on in this section. Is it only for users with local SMTP servers? I would add some context to prepare the user for the installation instructions: a brief paragraph at the top of this section explaining that we're installing the email plugin on a host with access to our SMTP server.